src/components/RequestsFilters/EffectiveLocationServicePointFilter/EffectiveLocationServicePointFilter.js

```diff
@@ -0,0 +1,74 @@
+import React, {
+  useCallback,
+  useMemo,
+} from 'react';
+import { isEmpty } from 'lodash';
+import PropTypes from 'prop-types';
+
+import {
+  Accordion,
+  FilterAccordionHeader,
+} from '@folio/stripes/components';
+import {
+  MultiSelectionFilter,
+} from '@folio/stripes/smart-components';
+
+import {
+  requestFilterTypes,
+} from '../../../constants';
+
+const EffectiveLocationServicePointFilter = ({
+  activeValues,
+  servicePoints,
+  onChange,
+  onClear,
+}) => {
+  const name = requestFilterTypes.EffLocation_SERVICE_POINT;
+
+  const clearFilter = useCallback(() => {
+    onClear(name);
+  }, [name, onClear]);
+
+  const filterOptions = useMemo(() => {
+    return servicePoints.map(servicePoint => ({
+      label: servicePoint.name,
+      value: servicePoint.id,
+    }));
+  }, [servicePoints]);
+
+  return (
+    <Accordion
+      data-testid="effectiveLocationServicePointAccordion"
+      displayClearButton={!isEmpty(activeValues)}
+      id={name}
+      header={FilterAccordionHeader}
+      label="Effective Location Primary Service Point Name"
```

Comment:
Could you please move label text to translation?
Could you please clarify label (current text not consistency)?